CHANGELOG.md

```diff
@@ -53,6 +53,8 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 * Configuration files now support [conditional
   variables](docs/config.md#conditional-variables).
 
+* Command aliases and default commands can now be defined as space-delimited strings.
```

Comment:
I wouldn't add this. We've overloaded some external command parameters, but that's for compatibility reasons.

Reply:
I don't see why not? We already support this for default-command, so why not for aliases as well?

Besides, I'd expect this to be possible coming from git.

Reply:
Also from my other comment:

> we could forbid quotes (", ' and `)

Reply:
ah yes, default-command works that way for compatibility reasons. Aliases doesn't have this problem.

> I don't see why not?

because it's not difficult to write a TOML array expression? A string version might also provide a false implication that it supports full shell syntax.